app/lib/services/get_an_identity/user_updater.rb

```diff
@@ -0,0 +1,34 @@
+module Services
+  module GetAnIdentity
+    class UserUpdater
+      def self.call(user:)
+        new(user:).call
+      end
+
+      attr_reader :user
+
+      def initialize(user:)
+        @user = user
+      end
+
+      def call
+        user = User.first
+        get_an_identity_user = ::GetAnIdentity::User.find(user.get_an_identity_id)
+
+        # GetAnIdentity::User#uid may not be the same as User#get_an_identity_id
+        # This is because while those two values both refer to the same thing,
+        # in cases of deduping we may get a new uid from GetAnIdentity when the user we
+        # looked up has been merged into another.
+        # By setting that new UID on the user, we can replicate that result within NPQ
+        user.update!(
```

Comment:
Just a thought is there any chance of us ever needing a record of the previous data? Should we create a history record for that data?

Reply:
Getting a gem installed so we can have an audit trail of record changes would be a really good idea, ECF has that I think. We should pitch it and see if there're any implications re: data protection.

Reply:
Yeah or other option short term is create a small history table to store the data before its changed. to be honest if this data is already captured maybe we dont need to worry?